tm2/pkg/bft/privval/signer/remote/client/errors.go

```diff
@@ -0,0 +1,21 @@
+package client
+
+import "errors"
+
+// Errors returned by the remote signer client.
```

Comment:
Don't let Morgan see this file, otherwise you are gonna hear a knock at the door :)

Reply:
What should I do to make Morgan happy then? :)

Reply:
Really a nitpick, but moving the errors to the file they originate from (where they're used)